govcd/adminvdc.go

```diff
@@ -23,6 +25,174 @@ func NewAdminVdc(cli *Client) *AdminVdc {
 	}
 }
 
+// vdcVersionedFuncs holds interface of functions
```

Comment:
```suggestion
// vdcVersionedFuncs is a generic representation of a VDC creation across multiple versions
```

Reply:
fixed

Reply:
@dataclouder "VDC creation" is too specific in this case, because tries to cover all CRUD operations:
![Screenshot 2020-02-13 at 14 07 24](https://user-images.githubusercontent.com/11330689/74434008-33284b00-4e6a-11ea-85ee-fbc913dceae8.png)
 What about changing "a VDC creation" to "VDC CRUD operations"? That would read:
`// vdcVersionedFuncs is a generic representation of VDC CRUD operations across multiple versions`

Reply:
Yes, it's better. Although we only support creation and update, not all CRUD, since `D` is missing and the `R` was explicitly implemented with hidden version choice (see my note under `GetAdminVDCByHref` ) instead of one function in this structure

Reply:
updated